.github/workflows/helper/configureV8.js

```diff
@@ -0,0 +1,61 @@
+fs = require('fs');
+
+function patchFile(path, find, replace) {
+  fs.readFile(path, 'utf8', function (err, data) {
+    if (err) {
+      return console.log(err);
+    }
+    data = data.replace(find, replace);
+
+    fs.writeFile(path, data, function (err) {
+      if (err) {
+        return console.log(err);
+      }
+    });
+  });
+}
+
+patchFile(
+  'app/android/app/build.gradle',
+  'enableHermes: true,',
+  'enableHermes: false,'
+);
+
+const mainApplicationPath =
+  'app/android/app/src/main/java/com/app/MainApplication.java';
+patchFile(
```

Comment:
It looks better with a newline here.
```suggestion
const mainApplicationPath =
  'app/android/app/src/main/java/com/app/MainApplication.java';

patchFile(
```